### DISCUSSION TITLE
I am developing my own downloader based on python so i have few question related to it

### DISCUSSION THREAD

**Discussion by Rachit-Pandey-2004** [Shabinder/SpotiFlyer#587] [Q&A]
First question what is your basic concept of downloading data from spotify or ganna .com.  I mean are you directly downloading from respective website or getting data from these website and downloading through youtube....

**Comment by Shabinder** [comment]
https://github.com/Shabinder/SpotiFlyer#note

**Reply by Arttzkie** [reply]
Mr.Shabinder can you please add spotify API so i can download the songs that I like to listen

**Comment by Rachit-Pandey-2004** [comment]
and then how are you extracting data from ganaa and jiosavaan about the artist and song name

**Reply by Shabinder** [reply]
Using Api's or Scraping
